src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -17,7 +16,7 @@ public sealed partial class PaneHolderPage : Page, IPaneHolder, ITabItemContent
 	{
 		public static event EventHandler<PaneHolderPage>? CurrentInstanceChanged;
 
-		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
+		private IUserSettingsService UserSettingsService { get; }
```

Comment:
```suggestion
		private IUserSettingsService userSettingsService { get; }
```

Reply:
According to the guideline section 2, item 1

> Use PascalCase for properties and static fields/properties